ntrans_gui.py

```diff
@@ -200,29 +236,42 @@ def generate_ntrans_dictionary(self) -> None:
             "target_language": self.target_language_var.get(),
         }
 
-        for key in user_choices:
-            if not user_choices[key]:
-                print(
-                    "All fields need to be filled in before generating the N-Trans dictionary."
-                )  # TODO: Change print to label
-                break  # TODO: break needs to change to `return`. Will be fixed in #23
+        for value in self.user_choices.values():
+            if not value:
+                self.setting_not_defined_warning.grid()
+                return
```

Comment:
This works, although it might be simpler written with a condition like `if all(self.user_choices.values())`. The `all` function returns `True` if every element of whatever you give as argument is true, and `False` otherwise.